metpy/calc/indices.py

```diff
@@ -202,3 +202,100 @@ def bulk_shear(pressure, u, v, heights=None, bottom=None, depth=None):
     v_shr = v_layer[-1] - v_layer[0]
 
     return u_shr, v_shr
+
+
+@exporter.export
+def supercell_composite(mucape, esrh, ebwd):
```

Comment:
A little too short on some names.
`esrh` -> `effective_storm_helicity`
`ebwd`-> `effective_bulk_shear`

Need to update names in docs as well.